frontend_app_kanban/src/component/KanbanCard.styl

```diff
@@ -37,3 +37,17 @@ cardWidth = 280px
 
     &__description
       overflow-wrap break-word
+      display flex
+      flex-direction column
+      & > div
+        line-height standardFontSize
+      &.kanban__contentpage__wrapper__board__card__description__overflow > div
+        // INFO - GB - 2021-12-21 - 45px is 3 * standardFontSize (15px)
+        max-height 45px
```

Comment:
You can do maths in Stylus
See [this section](https://stylus-lang.com/docs/operators.html#multiplicative---).
I'd suggest to remove the commentary and replace
``max-height 45px`` by ``max-height 3 * standardFontSize`` (or even ``max-height (3 * standardFontSize)`` to keep consistency with divide operator)
We already do calculation in stylus with `standardFontSize` in `NotificationWall.styl`